src/main/java/frc/robot/subsystems/ShooterHood.java

```diff
@@ -63,4 +72,20 @@ public void setStopPosition(boolean stopped) {
       }
     }
   }
+
+  /**
+   * Updates hood position LEDs
+   * 
+   * @param led LED to illuminate
+   */
+  public void setLEDs(HoodPosition position) {
```

Comment:
Why is this in the subsystem if it doesn't integrate with anything else? Why not just have the command set the LEDs?